modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -2388,7 +2388,7 @@ private boolean shouldBeImplicitHeader(CodegenParameter parameter) {
     @Override
     public void addImportsToOneOfInterface(List<Map<String, String>> imports) {
         if (additionalProperties.containsKey(JACKSON)) {
-            for (String i : Arrays.asList("JsonSubTypes", "JsonTypeInfo")) {
+            for (String i : Arrays.asList("JsonSubTypes", "JsonTypeInfo", "JsonIgnoreProperties")) {
```

Comment:
@wing328 FYI as this will affect _all_ Java codegen.

This is copying the behaviour that already exists in Java client though

Reply:
I removed the overrides from JavaClient and JavaHelidon, which were duplicates of this exact implementation. I presume this is what led to the bug in the first place - someone edited only the override in specialization and forgot about the parent method.